src/models/parallelCoordinates.js

```diff
@@ -157,14 +220,15 @@ nv.models.parallelCoordinates = function() {
         // simple options, just get/set the necessary values
         width:      {get: function(){return width;}, set: function(_){width=_;}},
         height:     {get: function(){return height;}, set: function(_){height=_;}},
-        dimensions: {get: function(){return dimensions;}, set: function(_){dimensions=_;}},
+        dimensionsName: { get: function() { return dimensionsName; }, set: function(_) { dimensionsName = _; } },
```

Comment:
Since your'e renaming the option, keep the dimension option there but log it to be deprecated.  See the tooltip and tooltipContent options in other charts for examples.